src/etl/variation_etl.py

```diff
@@ -76,14 +89,52 @@ class VariationETL(ETL):
             CREATE (gchrmn)-[ofc:ASSOCIATION]->(chrm)
             CREATE (gchrmn)-[ao:ASSOCIATION]->(a)
     """
+    notes_query_template = """
+           USING PERIODIC COMMIT %s
+           LOAD CSV WITH HEADERS FROM \'file:///%s\' AS row
+
+           MATCH (o:Variant {primaryKey:row.variantId})
+
+           CREATE (n:Note {primaryKey:row.noteId})
+                SET n.note = row.note          
+           MERGE (o)-[pn:ASSOCIATION]-(n)
```

Comment:
I am not sure but I think the pn variable is not necessary. Same thing with the next query